deploy/k8s/charts/trustification/templates/services/report/060-Deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if and .Values.modules.report.enabled }}
+{{- $mod := dict "root" . "name" "report-server" "component" "report" "module" .Values.modules.report -}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "trustification.common.name" $mod }}
+  labels:
+    {{- include "trustification.common.labels" $mod | nindent 4 }}
+  annotations:
+    {{- include "trustification.application.annotations" $mod | nindent 4 }}
+
+spec:
+  replicas: {{ include "trustification.application.replicas" $mod }}
+  selector:
+    matchLabels:
+      {{- include "trustification.common.selectorLabels" $mod | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        report: "server"
+        {{- include "trustification.common.selectorLabels" $mod | nindent 8 }}
+        {{- include "trustification.application.podLabels" $mod | nindent 8 }}
+
+    spec:
+      {{- include "trustification.application.pod" $mod | nindent 6 }}
+
+      containers:
+        - name: report
+          {{- include "trustification.common.defaultImage" $mod | nindent 10 }}
+          {{- include "trustification.application.container" $mod | nindent 10 }}
+
+          command: [ "/usr/sbin/nginx" ]
+          args: [ "-g", "daemon off;" ]
+          ports:
+            - containerPort: 8080
+              name: endpoint
+              protocol: TCP
+
+          volumeMounts:
+            - name: wwwdata
+              mountPath: /opt/app-root/src
+
+            - name: config-volume
+              mountPath: /etc/nginx/nginx.conf
+              subPath: nginx.conf
+              readOnly: true
+
+            {{- if .Values.report.auth.enabled }}
+            - name: auth-secret-volume
```

Comment:
As the schema now says "ValueOrRef", that needs to be implemented here. I guess it could follow a pattern like this: https://github.com/trustification/trustification/blob/bdf9fca358a71b7e22be155c19b7f9955c311d07/deploy/k8s/charts/trustification/templates/helpers/_auth.tpl#L1-L19